README.md

```diff
@@ -584,55 +566,58 @@ meant to be able to change with it.
     end
     ```
 
-  * <a name="let-and-let!"></a>
-    Use `let` and `let!` blocks instead of assigning values to instance
-    variables in `before` blocks.
-    <sup>[[link](#let-and-let!)]</sup>
+  * <a name="let-blocks"></a>
+    Use `let` and `let!` for data that is used across several examples in an
+    example group. Use `let!` to define variables even if they are not
+    referenced in examples, e.g. when testing negative cases.
+    Do not overuse `let`s for primitive data, find the balance between frequency
+    of use and complexity of the definition.
+    <sup>[[link](#let-blocks)]</sup>
 
     ```ruby
     # bad
-    describe '#type_id' do
-      before do
-        @resource = FactoryBot.create(:device)
-        @type = Type.find(@resource.type_id)
-      end
+    it 'finds entry' do
+      path = Path.new(1 => 2, 2 => 3)
+      expect(pathfinder.find_entry(path)).to eq(1)
+    end
 
-      it 'sets the type_id field' do
-        expect(@resource.type_id).to equal(@type.id)
-      end
+    it 'finds exit' do
+      path = Path.new(1 => 2, 2 => 3)
```

Comment:
The really important point of the description is

> find the balance between frequency of use and complexity of the definition.

I would not say that `path = Path.new(1 => 2, 2 => 3)` is sufficient duplication or complexity to call this example “bad”.

Reply:
Makes total sense. Added complexity to the shared object definition.